src/utils/searchUtils.ts

```diff
@@ -58,18 +59,29 @@ export function getSearchResultsHeading(
 }
 
 function buildQueryDisplayString(searchParams: SearchParams): string {
-  const searchFields = textInputFields.concat([
-    { name: "journal_title", label: "Journal Title" },
-    { name: "standard_number", label: "Standard Number" },
-    { name: "creatorLiteral", label: "author" },
-  ])
+  const searchFields = advSearchFields
+    // Lowercase the adv search field labels:
+    .map((field) => ({ ...field, label: field.label.toLowerCase() }))
+    .concat([
+      { name: "journal_title", label: "journal title" },
+      { name: "standard_number", label: "standard number" },
+      { name: "creatorLiteral", label: "author" },
+      { name: "oclc", label: "OCLC" },
+      { name: "isbn", label: "ISBN" },
+      { name: "issn", label: "ISSN" },
+      { name: "lccn", label: "LCCN" },
+    ])
   const paramsStringCollection = {}
-  const searchParamsObject = { ...searchParams, ...searchParams.filters }
+  const searchParamsObject = {
+    ...searchParams,
+    ...searchParams.filters,
+    ...searchParams.identifiers,
+  }
 
   Object.keys(searchParamsObject).forEach((param) => {
     const displayParam = searchFields.find((field) => field.name === param)
     if (displayParam && searchParamsObject[param]) {
-      let label = displayParam.label.toLowerCase()
```

Comment:
This was moved earlier since we only want to apply lowercase to the labels borrowed from the adv search fields config. For other things, we can directly control case in this function